packs/orion/actions/orion_health.py

```diff
@@ -0,0 +1,30 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from lib.actions import OrionBaseAction
+
+
+class OrionHealth(OrionBaseAction):
+    def run(self, platform):
+        """
+        Show details on an Orion monitoring platform health.
+        """
+
+        self.connect(platform)
+
+        # Does nothing yet...
```

Comment:
Yes, the PR started to get a little large so I ended up not getting further than the .py and test files. There is no YAML for the action so it'll not get in the way (plus the tests pass).

I'll complete this in another PR, if that's ok. As it's just a little complex...